internal/tui/components/dialogs/models/models.go

```diff
@@ -99,16 +99,8 @@ func (m *modelDialogCmp) Init() tea.Cmd {
 	providers, err := config.Providers()
 	if err == nil {
 		filteredProviders := []catwalk.Provider{}
-		simpleProviders := []string{
-			"anthropic",
-			"openai",
-			"gemini",
-			"xai",
-			"groq",
-			"openrouter",
-		}
 		for _, p := range providers {
-			if slices.Contains(simpleProviders, string(p.ID)) {
+			if strings.HasPrefix(p.APIKey, "$") && p.ID != catwalk.InferenceProviderAzure {
```

Comment:
most of this func is the same as `splash.go`'s `SetOnboarding`

Reply:
I will move this inside of the list.